octoprint_enclosure/BME680.py

```diff
@@ -0,0 +1,86 @@
+import bme680
+import time
+
+
+try:
+    sensor = bme680.BME680(bme680.I2C_ADDR_PRIMARY)
+except IOError:
+    sensor = bme680.BME680(bme680.I2C_ADDR_SECONDARY)
+
+hum_weighting = float(0.25)   # so hum effect is 25% of the total air quality score
+gas_weighting = float(0.75)   # so gas effect is 75% of the total air quality score
+
+sensor.set_humidity_oversample(bme680.OS_2X)
+sensor.set_pressure_oversample(bme680.OS_2X)
+sensor.set_temperature_oversample(bme680.OS_2X)
+sensor.set_filter(bme680.FILTER_SIZE_3)
+
+sensor.set_gas_heater_temperature(320)
+sensor.set_gas_heater_duration(150)
+sensor.select_gas_heater_profile(0)
+sensor.set_gas_status(bme680.ENABLE_GAS_MEAS)
+
+gas_reference = float(250000)
+hum_reference = float(40)
+getgasreference_count = int(0)
+
+
+def GetGasReference(gas_reference):
+  # Now run the sensor for a burn-in period, then use combination of relative humidity and gas resistance to estimate indoor air quality as a percentage.
```

Comment:
the indentation for this file is not consistent and should be 4 spaces everywhere.